primitives/src/v1/mod.rs

```diff
@@ -633,12 +638,12 @@ pub fn check_candidate_backing<H: AsRef<[u8]> + Clone + Encode>(
 		if sig.verify(&payload[..], &validator_id) {
 			signed += 1;
 		} else {
-			return Err(())
+			return Err(());
 		}
 	}
 
 	if signed != backed.validity_votes.len() {
-		return Err(())
+		return Err(());
```

Comment:
There is quite a bit of formatting bullshit in here, which hints that the usage of fmt either changed in this PR or the CI container bumped the nightly while this PR was open. Double check.